### ISSUE TITLE
Color option for clothing.

### ISSUE THREAD

**Issue by Kimax89** [ArmaLife/Framework#637]
I think we should have the option to choose colors/skins on clothing, just like we are able to with vehicles.

i believe this would help alot of server owners out there, in many different ways.

Example: i have 3 different skins for my cops and as the system works now, i have 3 different models and then added texture to each model.

**Comment by sargken** [comment]
I agree that would be awesome, you could do the same for Weapons as some weapons may have different color schemes?

**Comment by BoGuu** [comment]
It would require, in the way you've suggested it, some database editing, to record the colour along with the classname. For the cops, we already have an option for separate skins based on rank.

Of course this is entirely possible, and really not too difficult, but is it something that belongs in the Framework?

**Comment by tkcjesse** [comment]
@Kimax89 I've told you and others who have brought this up how to do it.. Pretty sure it was basically how BoGuu just mentioned that I suggested too. Just set a number with the item when it saves. 1-3 on item classname.

I don't think this is something that should be added since we don't have custom textures in now. Also, there isn't color variations on uniforms currently like there is on vehicles is there?

Not something that needs to be added imo...

**Comment by sargken** [comment]
Yes there is there is like 3 different patters on a Polo Shirt and so forth. I think its something that should be added.

**Comment by tkcjesse** [comment]
What? What polo shirt has 3 hidden textures??

U_C_Poloshirt_stripped, U_C_Poloshirt_redwhite, U_C_Poloshirt_salmon are not all just 1 polo?

**Comment by Kimax89** [comment]
or simular to how the carshop works with colors.

@BoGuu in not thinking about the cops right now, tho my cops have the chance to choose theyre uniform and not be restricted to the one provided by theyre rank.

I believe this could be considered an improvement and give the servers the ablilty to stand out from each other in the offers of skins/textures.
